docs/docs/recce-cloud/setup-gh-codespaces.md

```diff
@@ -54,35 +54,34 @@ GitHub Codespaces is a development environment provided by GitHub that allows de
     The GitHub token [generated by codespace](https://docs.github.com/en/codespaces/developing-in-a-codespace/default-environment-variables-for-your-codespace) is sufficient for Recce's use. It's not necessary to configure the `GITHUB_TOKEN` separately.
 
 ## How to use
-
-Here, we [use GitHub Codespaces for pull requests](https://docs.github.com/en/codespaces/developing-in-a-codespace/using-github-codespaces-for-pull-requests)
-
-1. Go to the GitHub PR page that is going to be reviewed
-1. Click the **Code** button and create a Codespace on the PR branch
-   ![alt text](../../assets/images/recce-cloud/setup-codespaces-pr.png){: .shadow}
-1. Starting a Codespace instance usually takes one to several minutes.
-1. When the Recce instance starts, an **Open Browser** notification will appear. Click the button to open the Recce web UI in a new tab.
-1. Or you can go to the **PORTS** tab to open the page.
-
-
+Once you complete [Recce Cloud setup](../../recce-cloud/#sign-up-the-recce-cloud), you can launch GitHub Codespaces from Recce Cloud's pull request page, and once it is started, the Recce instance will run and port forwarding will be set up.
+ 
+1. Go to Recce Cloud and click the repository you want to make changes. 
+    ![Recce Cloud Home](../../assets/images/recce-cloud/recce-cloud-home.png){: .shadow}
+2. Click the pull request that you want to use Recce instance.  
+    ![Recce Cloud Open PR](../../assets/images/recce-cloud/recce-cloud-open-pr.png){: .shadow}
+3. Click "Create in GitHub Codespaces."
+    ![Create in GitHub Codespaces](../../assets/images/recce-cloud/create-in-codespace.png){: .shadow}
+4. The Codespaces creation may take 1 to more than 5 mintues depeding on your Codespaces settings. And the Recce instance should take less than 1 minute to launch. 
+    - Please view [FAQ](../../recce-cloud/setup-gh-codespaces/#faq) for how to speed up.
+5. You can see the "State" of the progress; And the Action you can take in each state.  (need Andy's support )
```

Comment:
Hi Karen, I added some comments for states here. 

- Codespace Queued: the Codespace is creating

- Codespace Provisioning: the Codespace is provisioning

- Codespace Available: the Codespace is ready

- Recce launching: Recce instance is launching
    - Stop: stop launching Recce instance

- Recce active: Recce instance is launced successfully.
    - Open: open the Recce instance
    - Stop: stop the Recce instance (andy: it's stopping the codespace)
  
- Codespace ShuttingDown: the Codespace is shutting down

- Stopped: the Codespace is stopped and the Recce instance is closed.
    - Restart: restart the Codespace
    - Delete: delete the Codespace
        - It’s recommended to delete the Codespace once your PR is merged. (andy: delete the codespace when the PR is ready to be merged. Because when PR is merged, the PR page will not be shown anymore.)

Reply:
thanks! update them all. 
expect the last one, 
" delete the codespace when the PR is ready to be merged. Because when PR is merged, the PR page will not be shown anymore."
but the PR page URL is still there, e.g. https://cloud.datarecce.io/DataRecce/jaffle_shop_duckdb/pulls/32

Reply:
It's a bit tricky.
In the PR page, like https://cloud.datarecce.io/DataRecce/jaffle_shop_duckdb, you don't see the closed PR.

But if you use the PR URL or compose it yourself, Recce Cloud doesn't block it right now.

I think in the document, we suggest that users delete it if it's not used for the cost reason
And the UX can be followed in another story e.g., DRC-643 ?

Reply:
yes.